packages/xo-server-audit/src/index.js

```diff
@@ -1,11 +1,13 @@
-import appConf from 'app-conf'
 import asyncIteratorToStream from 'async-iterator-to-stream'
 import createLogger from '@xen-orchestra/log'
 import path from 'path'
+import TOML from '@iarna/toml'
 import { alteredAuditRecord, missingAuditRecord } from 'xo-common/api-errors'
 import { createGzip } from 'zlib'
 import { fromCallback } from 'promise-toolbox'
+import { merge } from 'lodash'
```

Comment:
Include specific function directly if you can, this remove a lot of code dependency:

```suggestion
import merge from 'lodash/merge'
```